core/build.gradle

```diff
@@ -25,7 +25,7 @@ dependencies {
     testImplementation 'org.hamcrest:hamcrest-library:1.3'
 }
 
-sourceCompatibility = 1.7
+sourceCompatibility = 1.8
```

Comment:
At the moment, we're still targeting JDK 7/Android 6.0. We have talked about raising this to something like JDK 8/Android 7.0 for version 0.17, but it's not decided yet.